app-containers/conmon/conmon-2.1.11.ebuild

```
@@ -0,0 +1,49 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="An OCI container runtime monitor"
+HOMEPAGE="https://github.com/containers/conmon"
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/conmon.git"
+else
+	SRC_URI="https://github.com/containers/conmon/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="+seccomp systemd"
+RESTRICT="test"
+
+RDEPEND="dev-libs/glib:=
+	seccomp? ( sys-libs/libseccomp )
+	systemd? ( sys-apps/systemd:= )"
+DEPEND="${RDEPEND}"
+BDEPEND="dev-go/go-md2man"
+
+src_prepare() {
+	# PR 505 https://github.com/containers/conmon/pull/505
+	echo "${PV}" | tee VERSION || die
```

Why not

```suggestion
	echo "${PV}" > VERSION || die
```

otherwise, we will have a stray 2.1.11 in the build log, no?

---

Note that the use of a bash pipe would also require using `assert` instead of `die`

---

> will have a stray 2.1.11 in the build log

that's exactly what I want. Intended. So I would know that change applied and it is temp change which should be removed in next version. 

> using assert instead of die

TIL. will do that next time